### DISCUSSION TITLE
Logging via AURA and Scenarios

### DISCUSSION THREAD

**Discussion by john-butn** [jongpie/NebulaLogger#540] [Q&A]
Hi All,

We are playing around with Nebula and community/experience sites. In particular with the situation where we need to raise the level for a particular set of users to debug an issue. 

Our org default (Log Level: INFO) is set as below:
![image](https://github.com/jongpie/NebulaLogger/assets/113883142/c18c876f-8602-4c55-9e10-72011f332a47)

We have created a profile specific setup for the community users (logged in) with an elevated log level (DEBUG):
![image](https://github.com/jongpie/NebulaLogger/assets/113883142/78b9f9c5-44c9-4c4a-a0cc-b08a88199241)

Within our aura components, in addition to the markup, have added the following in one of the 'doInit' and another method;
![image](https://github.com/jongpie/NebulaLogger/assets/113883142/5294cde8-11c3-4f9b-8666-c04d3920c205)

The logs show within the console:
![image](https://github.com/jongpie/NebulaLogger/assets/113883142/9ddacfad-a130-4a9a-acf6-f49529e8e280)

But no logs in the event stream are being generated. We have investigated the debug logs and no errors have surfaced, the only thing in the logs that gives any hint is the scenario appears to be set to null. I have tried everything I can think of - adding scenarios to the scenario table, settings, etc without success. I havent been able to find a reference in the debug logs to saveLog() being called.

Questions:
- Have a misunderstood the use/config of profile specific setups and scenarios?
- Has anyone encountered this before?
- Any suggestions as to what I should be checking?

Apologies if I have missed this in the discussions posts with my searches.

**Comment by stefanocortellessa** [comment]
We faced a very similar problem that I reported in the following issue: #556 

Were you able to resolve the issue?

**Comment by jongpie** [comment]
Hi @john-butn! Offhand, everything seems fine in your screenshots. One option you can try is to (temporarily) update the "Save Method" setting to "SYNCHRONOUS_DML" (instead of "EVENT_BUS"), and then see if any Apex exceptions happen when `saveLog()` is called.

You also mentioned that you added scenarios to the scenario table - are you referring the custom metadata type `LoggerScenarioRule__mdt`, or the custom object `LoggerScenario__c`?
